<launch>

  <include file="$(find novatel_gps_driver)/launch/tester_for_usb.launch"/>
  <node pkg="robot_localization" type="navsat_transform_node" name="navsat_transform_node" respawn="true">

    <param name="magnetic_declination_radians" value="0"/>

    <param name="yaw_offset" value="0"/>

    <remap from="/imu/data" to="/imu_data" />
    <remap from="/gps/fix" to="/fix" />
    <remap from="/odometry/filtered" to="/odom" />

  </node>

</launch>
